src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -238,6 +265,20 @@ export const MyVideosView = () => {
           <>
             <TabsContainer>
               <Tabs initialIndex={0} tabs={mappedTabs} onSelectTab={handleSetCurrentTab} />
+              <SortContainer>
+                <Text variant="body2">Sort by</Text>
+                <Select
+                  helperText={false}
+                  value={sortVideosBy}
+                  items={[...SORT_OPTIONS]}
```

Comment:
TS doesnt like it

Reply:
If we remove `as const` it shouldn't complain?

Reply:
That would make TS show the type of the array as `{
    name: string;
    value: VideoOrderByInput;
}[]` which would not give type safety for things we do like `typeof SORT_OPTIONS[number]` as any string/VideoOrderByInput combination would be valid instead we want that only the literals defined in the array to be valid + TS would allow the array to be mutable